ui/src/authProvider.js

```diff
@@ -99,9 +99,10 @@ const removeItems = () => {
 }
 
 const clearServiceWorkerCache = () => {
-  caches.keys().then(function (keyList) {
-    for (let key of keyList) caches.delete(key)
-  })
+  window.caches &&
```

Comment:
Just putting (caches&&...) threw up error in chrome android. 
window.caches works fine tho.